test/fixtures/cookbooks/wrapper/Berksfile

```diff
@@ -0,0 +1,10 @@
+source 'https://supermarket.chef.io'
+
+group :integration do
+  cookbook 'apt'
+  cookbook 'yum'
+end
+
+cookbook 'elkstack', :path => '../../../../'
```

Comment:
Tested it and you're right. It picks up changes in the local elkstack it's embedded in.